code/ai/aibig.cpp

```diff
@@ -1568,8 +1567,7 @@ void ai_big_strafe_attack()
 	accelerate_ship(aip, accel);
 
 	// if haven't been hit in quite a while, leave strafe mode
-	fix long_enough;
-	long_enough = F1_0 * STRAFE_MAX_UNHIT_TIME;
+	fix long_enough = F1_0 * fl2f(The_mission.ai_profile->strafe_max_unhit_time);
```

Comment:
This shouldn't be a float to fix should it? the `F1_0` multiplier ensures the result will be in the fix range. If this is meant to be 20 seconds, it should just be `F1_0 * The_mission.ai_profile->strafe_max_unhit_time` similar to as it was before, or alternatively just `fl2f(The_mission.ai_profile->strafe_max_unhit_time)` with no `F1_0`

Reply:
Good point, changed!